src/hbbft.erl

```diff
@@ -523,6 +534,8 @@ deserialize(M0, SK) ->
                                     end, DecShares),
                 sig_shares=maps:map(fun(_, Share) -> hbbft_utils:binary_to_share(Share, SK) end, SigShares),
                 thingtosign=NewThingToSign,
+                failed_combine=maps:get(failed_combine, M, []),
```

Comment:
is the worst case here that we retry the combine post-upgrade?

Reply:
not even that, this is just a tracking var